src/main/resources/data/minecolonies/crafterrecipes/sifter/gravel.json

```diff
@@ -7,5 +7,16 @@
         "count": "1"
     }
   ],
+  "additional-output": [
```

Comment:
These lists of possible outputs are NOT complete, they are a rough indicator of the sorts of things to be expected.  Some items require specific meshes, and some require certain luck/skill.

Reply:
I don't think it's a good idea to use additional-output for that, that's normally only for 100% guaranteed extra outputs (for conditional outputs, there's the loot table).

Reply:
I hear what you're saying, but the loot tables aren't easily readable for display in the UI, especially when nested. We also likely don't want to display some of the items due to the conditions involved.  We need a way to declaratively state this are possible outcomes, even though the code will NOT Produce anything from them without a primary output defined. (What's a secondary output without a primary?)

So far,this is the only scenario I found where we need a display only list of items for output, and adding another list to custom recipe and every recipeStorage seems like overkill, when we have two sitting idle in this scenario. I didn't want to use the alternate list, as it implies creation of a classic single-output recipe in the RS.